generator/webapp/src/test/java/io.fabric8.maven.generator.webapp/AppServerAutoDetectionTest.java

```diff
@@ -0,0 +1,107 @@
+package io.fabric8.maven.generator.webapp;
+
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Resource;
+import org.apache.maven.project.MavenProject;
+import org.junit.Test;
+
+import java.io.File;
+
+import static junit.framework.TestCase.*;
+
+/**
+ * @author kameshs
+ */
+public class AppServerAutoDetectionTest {
+
+    @Test
+    public void testIsWildFly() throws Exception {
+        String tmpDir = System.getProperty("java.io.tmpdir");
```

Comment:
An alternative would be to use `File.createTempFile()` or `Files.createTempDirectory()`. The advantage is that it gets cleaned up automatically after the test.